manifests/4.2/local-storage-operator.v4.2.0.clusterserviceversion.yaml

```diff
@@ -48,6 +48,7 @@ spec:
     - name: Source Repository
       url: https://github.com/openshift/local-storage-operator
   version: 4.2.0
+  minKubeVersion: 1.14.0
```

Comment:
Mirroring the same question here:

https://github.com/openshift/local-storage-operator/pull/73/files#r357030984